conductor/submitter_maya.py

```diff
@@ -225,6 +225,10 @@ class MayaConductorSubmitter(submitter.ConductorSubmitter):
     product = "maya-io"
 
     def __init__(self, parent=None):
+        
+        self.dependency_leaf_glob_file_path = None
```

Comment:
Why not instantiate here?

Reply:
Instantiate what?

Reply:
The env vars won't change during runtime, so these lines can be moved to the `__init__` safely:
```
        leaf_glob_file_path = os.environ.get('CONDUCTOR_DEPSCAN_LEAF_FILE', self.dependency_leaf_glob_file_path)
        exclude_glob_file_path = os.environ.get('CONDUCTOR_DEPSCAN_EXCLUDE_FILE', self.dependency_exclude_glob_file_path)
```
(also the default value for `.get()` will always be None as a result)

Reply:
True. Fixed.